core/src/loader.rs

```diff
@@ -53,7 +52,23 @@ pub fn load_system_program(info: &AccountInfo) -> Result<(), ProgramError> {
 /// * `Result<(), ProgramError>` - The result of the operation
 pub fn load_token_program(info: &AccountInfo) -> Result<(), ProgramError> {
```

Comment:
I think it makes sense for load_token_program to accept both `spl_token` and `token_2022` - or make another function that will accept both. For the most part we should be token program agnostic.

Reply:
Sounds great!